src/Umbraco.Core/Services/MemberService.cs

```diff
@@ -1022,25 +1062,29 @@ public void Save(IMember entity, bool raiseEvents = true)
                 }
 
                 uow.Commit();
-
+                
                 if (raiseEvents)
                 {
                     saveEventArgs.CanCancel = false;
                     uow.Events.Dispatch(Saved, this, saveEventArgs);
                 }
-            }
 
+                Audit(uow, AuditType.Save, "Save Member performed by user", userId, entity.Id);
+            }
+            
+            return OperationStatus.Success(evtMsgs);
         }
 
         /// <summary>
-        /// Saves a list of <see cref="IMember"/> objects
+        /// Saves a single <see cref="IMember"/> object
         /// </summary>
-        /// <param name="entities"><see cref="IEnumerable{IMember}"/> to save</param>
-        /// <param name="raiseEvents">Optional parameter to raise events.
-        /// Default is <c>True</c> otherwise set to <c>False</c> to not raise events</param>
-        public void Save(IEnumerable<IMember> entities, bool raiseEvents = true)
+        /// <param name="media">The <see cref="IMedia"/> to save</param>
+        /// <param name="userId">Id of the User saving the Media</param>
+        /// <param name="raiseEvents">Optional boolean indicating whether or not to raise events.</param>
+        Attempt<OperationStatus> IMemberServiceOperations.Save(IEnumerable<IMember> entities, int userId, bool raiseEvents)

```

Comment:
Looks like I probably introduced a breaking change here, need to correct it and redirect the old method to this new method.